isis/src/base/objs/Process/Process.h

```diff
@@ -240,9 +240,9 @@ namespace Isis {
       virtual void SetInputCube(Isis::Cube *inCube);
 
 
-      virtual Isis::Cube *SetOutputCube(const QString &parameter);
+      virtual Isis::Cube *SetOutputCube(const QString &parameter, UserInterface *ui=nullptr);
```

Comment:
This functional is overriden in ProcessMosaic, so it should be updated there too.